src/app/js/annotation/fields/ProposedValueFieldList.js

```diff
@@ -24,6 +24,7 @@ export function ProposedValueFieldList({
                 label={label}
                 options={options}
                 required
+                freeSolo={supportsNewValues}
```

Comment:
freeSolo is not very explicit, how about:
```suggestion
                supportsNewValues={supportsNewValues}
```